test/framework/universal_controlplane.go

```diff
@@ -168,6 +168,12 @@ func (c *UniversalControlPlane) GenerateDpToken(mesh, service string) (string, e
 }
 
 func (c *UniversalControlPlane) GenerateZoneIngressToken(zone string) (string, error) {
+	data := fmt.Sprintf(`'{"zone": "%s", "scope": ["ingress"]}'`, zone)
+
+	return c.generateToken("/zone", data)
+}
+
+func (c *UniversalControlPlane) GenerateZoneIngressLegacyToken(zone string) (string, error) {
```

Comment:
Do we already have an issue to remove these methods in 2 versions?

Reply:
https://github.com/kumahq/kuma/issues/5105